package.json

```diff
@@ -11,6 +11,8 @@
     "test": "ava",
     "lint-fix": "yarn lint --fix",
     "lint": "eslint 'snippets/**/*.js'",
+    "format": "prettier --write '**/*.md' --config .prettierrc.json",
+    "lint:format": "prettier --check '**/*.md' --config .prettierrc.json",
```

Comment:
To keep things from decaying, this has to be enforced in ci. So the `lint` script should call `lint:format`, I think. Or the ci workflow should be changed to call `lint:format`.

I suppose we can take care of that in a later PR, but here would be better.

Reply:
Already added in this PR. https://github.com/Agoric/documentation/blob/a7bb1e108b5b5a9dc2e5a9465906a1ba5837ac08/.github/workflows/test-markdown.yml#L29